sql-ballerina/Module.md

```diff
@@ -0,0 +1,302 @@
+    created for your database unless a connection pool matching with the properties you provided already exists. 
+    The JDBC module example below shows how the global connection pool is used. 
+
+    ```ballerina
+    jdbc:Client|sql:Error dbClient = 
+                               new ("jdbc:mysql://localhost:3306/testdb", 
+                                "root", "root");
+    ```
+
+2. Client owned, unsharable connection pool
+
+    If you define the `connectionPool` field inline when creating the database client with the `sql:ConnectionPool` type, 
+    an unsharable connection pool will be created. The JDBC module example below shows how the global 
+    connection pool is used.
+
+    ```ballerina
+    jdbc:Client|sql:Error dbClient = 
+                               new (url = "jdbc:mysql://localhost:3306/testdb", 
+                               connectionPool = { maxOpenConnections: 5 });
+    ```
+
+3. Local, shareable connection pool
+
+    If you create a record of type `sql:ConnectionPool` and reuse that in the configuration of multiple clients, 
+    for each  set of clients that connects to the same database instance with the same set of properties, a shared 
+    connection pool will be created. The JDBC module example below shows how the global connection pool is used.
+
+    ```ballerina
+    sql:ConnectionPool connPool = {maxOpenConnections: 5};
+    
+    jdbc:Client|sql:Error dbClient1 =       
+                               new (url = "jdbc:mysql://localhost:3306/testdb",
+                               connectionPool = connPool);
+    jdbc:Client|sql:Error dbClient2 = 
+                               new (url = "jdbc:mysql://localhost:3306/testdb",
+                               connectionPool = connPool);
+    jdbc:Client|sql:Error dbClient3 = 
+                               new (url = "jdbc:mysql://localhost:3306/testdb",
+                               connectionPool = connPool);
+    ```
+    
+#### Closing the Client
+
+Once all the database operations are performed, you can close the database client you have created by invoking the `close()`
+operation. This will close the corresponding connection pool if it is not shared by any other database clients. 
+
+```ballerina
+error? e = dbClient.close();
+```
+Or
+```ballerina
+check dbClient.close();
+```
+
+### Database Operations
+
+Once the client is created, database operations can be executed through that client. This module defines the interface 
+and common properties that are shared among multiple database clients.  It also supports querying, inserting, deleting, 
+updating, and batch updating data.  
+
+#### Creating Tables
+
+This sample creates a table with two columns. One column is of type `int` and the other is of type `varchar`.
+The CREATE statement is executed via the `execute` remote function of the client.
+
+```ballerina
+// Create the ‘Students’ table with the  ‘id’, 'name' and ‘age’ fields.
```

Comment:
```suggestion
// Create the ‘Students’ table with the  ‘id’, 'name', and ‘age’ fields.
```

Reply:
Added suggested change in https://github.com/ballerina-platform/module-ballerina-sql/pull/228/commits/fa55a74e7928c9529090430cd4a5a358e2af8e8e